clickhouse-admin/api/src/lib.rs

```diff
@@ -64,7 +64,7 @@ pub trait ClickhouseAdminKeeperApi {
     /// Retrieve configuration information from a keeper node.
     #[endpoint {
         method = GET,
-        path = "/keeper/conf",
+        path = "/conf",
```

Comment:
I do find it somewhat confusing that we have a `/conf` and a `/config` endpoint. Should we change this one to be `zk_conf`, and likewise change the other 4lwc to have a `zk_` prefix, since they are inherited from zookeeper? I think for now that's just `zk_lgif`.

Reply:
I also don't love having `/conf` and `/config`. I'm not fully sold on the `zk_` prefix as it kinda feels redundant too?
how about `4lw_` as the prefix?

Alternitavely, we could have the `/config` endpoint be `xml_config` and then leave these as is?

WDYT? I'm kinda leaning on the second option

Reply:
I also thought about both of those options. I don't like `/xml_config` because we don't actually send the data as XML and it's possible we decide to generate or clickhouse requires us to generate it in a different format inside the server. It's an implementation detail from the API POV. 

I really like `4lw_` the best. However, I'm unsure if we can use a number as the first character in an endpoint. If we can, I'd go with that. If not we could use `flw_`.

Reply:
Done!

```console
$ cargo run --bin=clickhouse-admin-keeper -- run -c ./smf/clickhouse-admin-server/config.toml -a [::1]:8888 -l [::1]:20001 -b /Users/karcar/src/omicron/out/clickhouse/clickhouse
   Compiling omicron-clickhouse-admin v0.1.0 (/Users/karcar/src/omicron/clickhouse-admin)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 2.73s
     Running `target/debug/clickhouse-admin-keeper run -c ./smf/clickhouse-admin-server/config.toml -a '[::1]:8888' -l '[::1]:20001' -b /Users/karcar/src/omicron/out/clickhouse/clickhouse`
note: configured to log to "/dev/stdout"
{"msg":"listening","v":0,"name":"clickhouse-admin-keeper","level":30,"time":"2024-11-07T21:56:28.764057Z","hostname":"ixchel","pid":42397,"local_addr":"[::1]:8888","component":"dropshot","file":"/Users/karcar/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dropshot-0.12.0/src/server.rs:197"}
{"msg":"accepted connection","v":0,"name":"clickhouse-admin-keeper","level":30,"time":"2024-11-07T21:56:32.72645Z","hostname":"ixchel","pid":42397,"local_addr":"[::1]:8888","component":"dropshot","file":"/Users/karcar/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dropshot-0.12.0/src/server.rs:1105","remote_addr":"[::1]:63616"}
{"msg":"Retrieved data from `clickhouse keeper-client --q lgif`","v":0,"name":"clickhouse-admin-keeper","level":30,"time":"2024-11-07T21:56:33.183627Z","hostname":"ixchel","pid":42397,"component":"ClickhouseCli","file":"clickhouse-admin/types/src/lib.rs:290","output":"\"first_log_idx\\t1\\nfirst_log_term\\t1\\nlast_log_idx\\t13399\\nlast_log_term\\t180\\nlast_committed_log_idx\\t13399\\nleader_committed_log_idx\\t13399\\ntarget_committed_log_idx\\t13399\\nlast_snapshot_idx\\t13393\\n\\n\""}
{"msg":"request completed","v":0,"name":"clickhouse-admin-keeper","level":30,"time":"2024-11-07T21:56:33.183798Z","hostname":"ixchel","pid":42397,"uri":"/4lw-lgif","method":"GET","req_id":"5a8b6051-7f4e-4d8b-8fad-95d2a5676eba","remote_addr":"[::1]:63616","local_addr":"[::1]:8888","component":"dropshot","file":"/Users/karcar/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dropshot-0.12.0/src/server.rs:950","latency_us":456717,"response_code":"200"}
```

```console
$ curl http://[::1]:8888/4lw-lgif
{"first_log_idx":1,"first_log_term":1,"last_log_idx":13399,"last_log_term":180,"last_committed_log_idx":13399,"leader_committed_log_idx":13399,"target_committed_log_idx":13399,"last_snapshot_idx":13393}
```